README.md

```diff
@@ -3,22 +3,22 @@
 This is the Intel staging area for llvm.org contributions and the home for
 Intel LLVM-based projects:
 
-- [oneAPI Data Parallel C++ compiler](#oneapi-data-parallel-c-compiler)
+- [oneAPI DPC++ compiler](#oneapi-dpc-compiler)
 - [Late-outline OpenMP and OpenMP Offload](#late-outline-openmp-and-openmp-offload)
```

Comment:
Do we not need this anymore?

Reply:
Short name have preference today

Reply:
I should have been clearer; I meant the OpenMP late-outline link.   That seems to have been removed.   Is that not needed anymore?

Reply:
Sorry, nevermind.   I was confused by the '-' used in the bullets.   I thought it was a diff line.